### PR TITLE
Fix hardcoded reference to cfn_outputs image

### PR DESCRIPTION
**Pull request by sshvans**
*Issue #, if available:*
cfn_outputs image is not linked via anchor tag, instead hardcoded.

*Description of changes:*
Fixed hardcoded reference to cfn_outputs image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: deployment_steps.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ include::../{generateddir}/parameters/index.adoc[]
 . On the *Review* page, review and confirm the template settings. Under *Capabilities*, select the two check boxes to acknowledge that the template creates IAM resources and might require the ability to automatically expand macros.
 . Choose *Create stack* to deploy the stack.
 . Monitor the status of the stack. When the status is *CREATE_COMPLETE*, the {partner-product-name} deployment is ready.
-. Use the values displayed in the *Outputs* tab for the stack, as shown in figure 2, to view the created resources.
+. Use the values displayed in the *Outputs* tab for the stack, as shown in <<cfn_outputs>>, to view the created resources.
 
 [#cfn_outputs]
```
Comment (vsnyc) [inline deployment_steps.adoc:22]: Could you add `:xrefstyle: short` before [#cfn_outputs]

Comment (sshvans) [reply deployment_steps.adoc:23]: Done!

**Comment by vsnyc** [comment]
Looks good to me. I've tested that it works. @andrew-glenn, can we merge this?

**Comment by andrew-glenn** [comment]
K. Thanks.

**Pull request merged**